envs/dev/values.yaml

```diff
@@ -2,4 +2,7 @@ spec:
   destination:
     server: https://kubernetes.default.svc
   source:
-    targetRevision: HEAD
\ No newline at end of file
+    repoURL: https://github.com/aws-samples/ssp-eks-workloads
+    targetRevision: main
+  ingress:
+    host:
```

Comment:
Is value supposed to be provided by the customer? If yes, how will the customer know about it, are you planning a section in the documentation somewhere?